Libplanet/Tx/Transaction.cs

```diff
@@ -268,180 +268,6 @@ public static Transaction<T> FromBencodex(byte[] bytes)
-        /// it can cover every case including &#x201c;rehearsal mode.&#x201d;
-        /// The <see cref="IActionContext.Rehearsal"/> property also might be
-        /// useful to make the <see cref="IAction"/> can deal with the case of
-        /// rehearsal mode.</para>
-        /// <para>The actual exception that an <see cref="IAction"/> threw
-        /// is stored in its <see cref="Exception.InnerException"/> property.
-        /// </para>
-        /// </exception>
-        public static Transaction<T> Create(
-            long nonce,
-            PrivateKey privateKey,
-            IEnumerable<T> actions,
-            IImmutableSet<Address> updatedAddresses = null,
-            DateTimeOffset? timestamp = null
-        )
-        {
-            if (ReferenceEquals(privateKey, null))
-            {
-                throw new ArgumentNullException(nameof(privateKey));
-            }
-
-            PublicKey publicKey = privateKey.PublicKey;
-            var signer = new Address(publicKey);
-
-            if (ReferenceEquals(updatedAddresses, null))
-            {
-                updatedAddresses = ImmutableHashSet<Address>.Empty;
-            }
-
-            DateTimeOffset ts = timestamp ?? DateTimeOffset.UtcNow;
-
-            ImmutableArray<T> actionsArray = actions.ToImmutableArray();
-            byte[] payload = new Transaction<T>(
-                nonce,
-                signer,
-                publicKey,
-                updatedAddresses,
-                ts,
-                actionsArray
-            ).ToBencodex(false);
-
-            if (!actionsArray.IsEmpty)
-            {
-                IAccountStateDelta delta = new Transaction<T>(
-                    nonce,
-                    signer,
-                    publicKey,
-                    updatedAddresses,
-                    ts,
-                    actionsArray
-                ).EvaluateActions(
-                    default(HashDigest<SHA256>),
-                    0,
-                    new AccountStateDeltaImpl(_ => null),
-                    signer,
-                    rehearsal: true
-                );
-                if (!updatedAddresses.IsSupersetOf(delta.UpdatedAddresses))
-                {
-                    updatedAddresses =
-                        updatedAddresses.Union(delta.UpdatedAddresses);
-                    payload = new Transaction<T>(
-                        nonce,
-                        signer,
-                        publicKey,
-                        updatedAddresses,
-                        ts,
-                        actionsArray
-                    ).ToBencodex(false);
-                }
-            }
-
-            byte[] sig = privateKey.Sign(payload);
-            return new Transaction<T>(
-                nonce,
-                signer,
-                publicKey,
-                updatedAddresses,
-                ts,
-                actionsArray,
-                sig
-            );
-        }
```

Comment:
Does this method have to be removed at all?  I thought this is fine to remain, and `BlockChain<T>.MakeTransaction()` method could be a thin façade on this.

Reply:
I think it's okay to leave this, but is there a use case?